src/legacy/store/GraphQLDeferredQueryTracker.js

```diff
@@ -17,7 +17,7 @@ var ErrorUtils = require('ErrorUtils');
 var Map = require('Map');
 import type {DataID} from 'RelayInternalTypes';
 import type RelayQuery from 'RelayQuery';
-var RelayStoreData = require('RelayStoreData');
+var RelayRecordStore = require('RelayRecordStore');
```

Comment:
`import type RelayRecordStore from 'RelayRecordStore'` - only import the type unless you'll use the value at runtime.